src/Udyam/components/Events/events.js

```diff
@@ -21,50 +21,68 @@ const data = [
+    psLink: '',
+    submissionLink: '',
+    registerLink: 'https://eesiitbhu.in/dashboard#events-register'
     psLink: 'https://devbits-web-development-hackathon.devfolio.co/',
     submissionLink: ''
   },
   {
     title: 'COMMNET',
     description: 'Become proficient in Network Architecture and MATLAB implementation of coding theory, compression algorithms, signal processing, modulation and demodulation techniques, designing and simulating analog circuits & filters to perform computations using CAD tools.',
     psLink: 'https://drive.google.com/file/d/1F_l-Xc7C3sYEp7azpT2bQWIU9-_JhH2O/view?usp=share_link',
-    submissionLink: ''
+    submissionLink: '',
+    registerLink: 'https://eesiitbhu.in/dashboard#events-register'
   },
   {
     title: 'X-IOT-A',
     description: 'Propose and implement a solution to a real-world problem with the help of the booming technology of the Internet of Things involving circuit designing and the incorporation of hardware and software.',
+    psLink: 'https://drive.google.com/file/d/1d3UkJB_tNFYfI2HBMR_T_DhQsF07eDfw/view?usp=share_link',
+    submissionLink: 'https://docs.google.com/forms/d/e/1FAIpQLSf0igPnY4bjQiIgp7EVZYhWiiQbkyBFxICFL5Kvz267avGwNQ/viewform',
+    registerLink: 'https://eesiitbhu.in/dashboard#events-register'
     psLink: 'https://drive.google.com/drive/folders/1zIsHDTuZ4gG8T5lJGv3NKnMyTCKchsra?usp=sharing',
     submissionLink: 'https://docs.google.com/forms/d/e/1FAIpQLSf0igPnY4bjQiIgp7EVZYhWiiQbkyBFxICFL5Kvz267avGwNQ/viewform'
   },
   {
     title: 'CASSANDRA',
     description: 'Prove yourself a good data analyst by designing a model that learns and optimizes the dataset provided, implementing Machine Learning algorithms in industrial problems and gain exposure to the ABCs of data science.',
     psLink: '',
-    submissionLink: ''
+    submissionLink: '',
+    registerLink: 'https://eesiitbhu.in/dashboard#events-register'
   },
   {
     title: 'MOSAIC',
     description: 'Get the opportunity to decipher canonical problems based on the intriguing subject of Machine Learning and Computer Vision. Know about deep learning and image processing inside out and acquire expertise in machine learning.',
+    psLink: 'https://drive.google.com/file/d/1IgzULOh1dQkfOVA9FQqoxO_9lvTPe0cT/view?usp=share_link',
+    submissionLink: '',
+    registerLink: 'https://eesiitbhu.in/dashboard#events-register'
     psLink: 'https://drive.google.com/drive/folders/1cR8pelM4daqMg530qPYvZqKcMk8fOx88?usp=share_link',
     submissionLink: ''
   },
   {
     title: 'FUNCKIT',
     description: 'Test your problem-solving abilities through low-level programming and building gate-level circuitry to optimize the logic and hardware and decrease execution time.',
     psLink: 'https://drive.google.com/file/d/1PAd6g6Qw9HTc3o9usjeKzlSbVzTqbZ0P/view?usp=share_link',
-    submissionLink: ''
+    submissionLink: '',
+    registerLink: 'https://eesiitbhu.in/dashboard#events-register'
```

Comment:
psLink and submissionLink needs to be defined only once in every event array. 
Add *,* after giving link of every variable

Reply:
Done